sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -594,17 +594,21 @@ private static void printVerdict(final long numberOfTestFailures, final File tes
         }
 
         if (numberOfTestFailures == 0) {
-            LOG.info(
-                    "Test run with {} Tests was completed successfully. No problems were found.",
-                    testRunObserver.getTotalNumberOfTestsRun());
+            if (testRunObserver.isInvalid()) {
+                LOG.info(
+                        "Test run with {} Tests was completed successfully but other problems were found. Please consult the logfiles in {} for further information.",
+                        testRunObserver.getTotalNumberOfTestsRun(),
+                        testRunDir);
+                LOG.info("Test run was invalid.");
+            } else {
+                LOG.info(
+                        "Test run with {} Tests was completed successfully. No problems were found.",
+                        testRunObserver.getTotalNumberOfTestsRun());
+                LOG.info("Test run was valid.");
+            }
         } else {
             LOG.info("Test run found problems. Please consult the logfiles in {} for further information.", testRunDir);
```

Comment:
```suggestion
            LOG.info("Test run with {} tests found problems. Please consult the logfiles in {} for further information.",
            testRunObserver.getTotalNumberOfTestsRun(),
             testRunDir);
```

or perhaps better after changing the nesting as requested:

```suggestion
            LOG.info("Test run with {} tests was valid, but found problems. Please consult the logfiles in {} for further information.",
            testRunObserver.getTotalNumberOfTestsRun(),
             testRunDir);
```

Reply:
done